src/Compiler/pars.fsy

```diff
@@ -5744,15 +5744,15 @@ recdExprCore:
   | appExpr WITH recdBinding recdExprBindings opt_seps_block
      { let l = List.rev $4
        let l = rebindRanges $3 l $5
-       (Some($1, $5), l) }
+       (Some($1, None), l) }
```

Comment:
The captured info is still different here.

Reply:
No. this should be `None` as previously was `BlockSeparator.Offside`